bin/seed-csv.js

```diff
@@ -0,0 +1,29 @@
+import commander from 'commander';
+import fs from 'fs';
+import csvm from '../app/models/CSVModels.js';
+import {rows} from '../app/models/CSVModels.js';
+
+const { program } = commander;
+
+program
```

Comment:
If you're not using args, then you can do `const args = process.argv.slice(2);` to get an array of values. I don't think you're using args at all in this script though, so you can drop it entirely.